p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -1682,8 +1657,9 @@ baseType
 
 ### The void type
 
-The void type is written as ```void```. It contains no values. It can
-only appear in few restricted places in P4 programs.
+The void type is written as ```void```. It contains no values. It is
+not included in the set of base types as it can only appear in few
```

Comment:
Saying here that void is not included in the set of base types seems at odds with void being included in the list above.